packages/api/package.json

```diff
@@ -11,7 +11,8 @@
   "contributors": [
     "Michael Clayton <mclayton@redhat.com>",
     "Kun Yan <kyan@redhat.com>",
-    "Sayak Sarkar <ssarkar@redhat.com >"
+    "Sayak Sarkar <ssarkar@redhat.com >",
+    "Soumyadip Chowdhury"
```

Comment:
@soumyadip007 can you please also add your email to the contributors list

Reply:
Sure Deepesh, I'll add it